### DISCUSSION TITLE
wild card for the search feature

### DISCUSSION THREAD

**Discussion by tible** [DaxStudio/DaxStudio#742] [Ideas]
Ciao friends,

As a DAX user, I feel the need of having a wild card when searching through tens of measures I build and have displayed only those that satisfy the search criteria so that I can select them easier when building my query.

e.g. 
I have the following measures created:
```
area.categ1.subcateg1
area.categ1.subcateg2
area.categ1.subcateg3
area.categ1.subcateg4
area.categ1.subcateg5
area.categ2.subcateg1
area.categ2.subcateg2
area.categ2.subcateg3
area.categ2.subcateg4
area.categ2.subcateg5
area.categ3.subcateg1
area.categ3.subcateg2
area.categ3.subcateg3
area.categ3.subcateg4
area.categ3.subcateg5

```

... and when searching for ```*3``` where the asterisc is the wild card, the result should be the following:

```
area.categ1.subcateg3
area.categ2.subcateg3
area.categ3.subcateg3
```

Grazie e ciao!
Tiberius

**Comment by dgosbell** [comment]
Currently the metadata search does a "contains" search, but it does not start searching until you enter 2 or more characters. So in this specific example you could search for "g3" and it will get the results you are after, but I can see that a wildcard search would be more flexible. I'll create an issue for this in the backlog to look at when I get a chance.
